globus_automate_client/cli/actions.py

```diff
@@ -14,6 +16,45 @@
 app = typer.Typer(short_help="Manage Globus Automate Actions")
 
 
+class ActionInputFormat(str, Enum):
+    json = "json"
+    yaml = "yaml"
```

Comment:
This is going to look out of order, but perhaps it will make more sense after seeing the comments related to DUMPERS and hypothetical LOADERS. Seems one could simply put the `load` method on this InputFormat Enum and DUMPERS on the OutputFormat Enum. Still could make use of the Dict approach to map to the real dump functions and just make that a class property.

Again, splitting hairs to some degree. I'm ok with any of the mentioned approaches including what's already in the code.